extensions/blocks/podcast-player/view.js

```diff
@@ -0,0 +1,55 @@
+/* global _wpmejsSettings, MediaElementPlayer, jetpackPodcastPlayers */
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+
+const playerInstances = {};
+const meJsSettings = typeof _wpmejsSettings !== undefined ? _wpmejsSettings : {};
+
+const initializeBlock = function( id ) {
+	const block = document.getElementById( id );
+	if ( block ) {
+		const player = {
+			id,
+			block,
+			audio: block.querySelector( 'audio' ),
+			currentTrack: 0,
+		};
+		playerInstances[ id ] = player;
+
+		// Initialize player UI.
+		player.mediaElement = new MediaElementPlayer( player.audio, meJsSettings );
```

Comment:
Should we tests for presence of `MediaElementPlayer` before we attempt to initialise any of this code? If not available we should render the default semantic HTML experience.

Reply:
ME is a core dependency, same as jQuery and not a single person tests for it being present. We "trust" WordPress I guess :D We can probably do an early return if ME not present. The HTML that's rendered it good enough to be used without JS

Reply:
ME is a core dependency, same as jQuery and not a single person tests for it being present. We "trust" WordPress on this, I guess.

We can probably do an early return if ME not present. The HTML that's rendered it good enough to be used without JS

Reply:
> We can probably do an early return if ME not present

I'd still say do the resilient coding bit and bale if not found. Extra protection!